careers.html

```diff
@@ -31,7 +30,7 @@
   <div class="team-overlay"></div>
   <div class="team-center">
       <p class="team-caption">BUILD future with us</p>
-      <p class="team-button">Join us</p>
+      <a class="team-button" id="js-join-scroll">Join us</a>
```

Comment:
@priyanka0901 We should use only `camelcase` for `ID` naming convention. @Swathi8 Can correct me if I am wrong.